### DISCUSSION TITLE
Any guide or list of achievements?

### DISCUSSION THREAD

**Discussion by mtorres-root** [community/community#18864] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm interested in knowing how to earn more achievements but I can't find any information. Thanks

**Comment by supermavster** [comment, marked as answer]
This link for a full list of achievements currently available on GitHub - https://github.com/Schweinepriester/github-profile-achievements

**Comment by lkant8** [comment]
This link for a full list of achievements currently available on GitHub - https://github.com/codewith-fun?tab=achievements

**Comment by lara-rium** [comment]
why isn't this listed officially on the profile? why are all achievements secret

**Comment by Andreaa-Dev** [comment]
You can easily get the Yolo by merge your own branch 😊. I tried and it worked !!!

**Reply by lara-rium** [reply]
already have it but i mean a list would be nice to see our progression
